test/test_transforms.py

```diff
@@ -135,72 +219,57 @@ def test_center_crop_2(self):
-            expected_output = (tl, tr, bl, br, center)
-            self.assertEqual(results, expected_output)
 
-    def test_ten_crop(self):
-        to_pil_image = transforms.ToPILImage()
-        h = random.randint(5, 25)
-        w = random.randint(5, 25)
-        for should_vflip in [True, False]:
-            for single_dim in [True, False]:
-                crop_h = random.randint(1, h)
-                crop_w = random.randint(1, w)
-                if single_dim:
-                    crop_h = min(crop_h, crop_w)
-                    crop_w = crop_h
-                    transform = transforms.TenCrop(crop_h,
-                                                   vertical_flip=should_vflip)
-                    five_crop = transforms.FiveCrop(crop_h)
-                else:
-                    transform = transforms.TenCrop((crop_h, crop_w),
-                                                   vertical_flip=should_vflip)
-                    five_crop = transforms.FiveCrop((crop_h, crop_w))
-
-                img = to_pil_image(torch.FloatTensor(3, h, w).uniform_())
-                results = transform(img)
-                expected_output = five_crop(img)
-
-                # Checking if FiveCrop and TenCrop can be printed as string
+def test_center_crop():
+    height = random.randint(10, 32) * 2
+    width = random.randint(10, 32) * 2
+    oheight = random.randint(5, (height - 2) / 2) * 2
+    owidth = random.randint(5, (width - 2) / 2) * 2
+
+    img = torch.ones(3, height, width)
+    oh1 = (height - oheight) // 2
+    ow1 = (width - owidth) // 2
+    imgnarrow = img[:, oh1:oh1 + oheight, ow1:ow1 + owidth]
+    imgnarrow.fill_(0)
+    result = transforms.Compose([
+        transforms.ToPILImage(),
+        transforms.CenterCrop((oheight, owidth)),
+        transforms.ToTensor(), ])(img)
+    assertEqual(result.sum(), 0,
+                "height: {} width: {} oheight: {} owdith: {}".format(height, width, oheight, owidth))
+    oheight += 1
+    owidth += 1
+    result = transforms.Compose([
+        transforms.ToPILImage(),
+        transforms.CenterCrop((oheight, owidth)),
+        transforms.ToTensor(), ])(img)
+    sum1 = result.sum()
+    assertGreater(sum1, 1,
+                  "height: {} width: {} oheight: {} owdith: {}".format(height, width, oheight, owidth))
+    oheight += 1
+    owidth += 1
+    result = transforms.Compose([
+        transforms.ToPILImage(),
+        transforms.CenterCrop((oheight, owidth)),
+        transforms.ToTensor(), ])(img)
+    sum2 = result.sum()
+    assertGreater(sum2, 0,
+                  "height: {} width: {} oheight: {} owdith: {}".format(height, width, oheight, owidth))
+    assertGreater(sum2, sum1,
+                  "height: {} width: {} oheight: {} owdith: {}".format(height, width, oheight, owidth))
+
+    def test_autoaugment():
+        for policy in transforms.AutoAugmentPolicy:
+            for fill in [None, 85, (128, 128, 128)]:
```

Comment:
We'll need to parametrize over policy and fill instead of having nested for loops